src/main/resources/migrations/00013_add_fee_columns_to_transaction_table.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_fee_columns_to_transaction_table
+
+ALTER TABLE transaction
+    ADD COLUMN fee BIGINT,
+    ADD COLUMN corporate_surcharge BIGINT;
```

Comment:
I think `corporate_surcharge` is currently stored in the `transaction_details` column on transactions. Do we need it at the top level too?

Reply:
No we don't need to - it would be one or another.
The story describes appropriate columns being available, and after writing the story there was a discussion to do it differently.

Reply:
Will remove `corporate_surcharge` from top level.